dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -27,6 +29,8 @@ const getGefFacilitiesController = require('../controllers/portal/gef-facility/g
 const createGefFacilityController = require('../controllers/portal/gef-facility/create-gef-facility.controller');
 const updateGefFacilityController = require('../controllers/portal/gef-facility/update-facility.controller');
 
+const getGefFacilityAmendmentController = require('../controllers/portal/facility/get-amendment.controller');
```

Comment:
Does this need GEF in the name, doesn't seem to be anything GEF specific? Looks like it should be `getFacilityAmendmentController`